src/components/globalConfigurations/GlobalConfiguration.tsx

```diff
@@ -446,13 +446,11 @@ function Body({ getHostURLConfig, checkList, serverMode, handleChecklistUpdate,
                 path={URLS.GLOBAL_CONFIG_CLUSTER}
                 render={(props) => {
                     return (
-                        <div className="flexbox">
```

Comment:
I have you have verified the impact of this

Reply:
Yes there is no imapcet have confirmed with the testing. Earlier page loader & empty state was stick at the top.